### DISCUSSION TITLE
Is there a way to export app outside of backend?

### DISCUSSION THREAD

**Discussion by gmentzik** [Budibase/budibase#10238] [Help] [answered]
Hi devs,
Regarding self host Budibase, is there a way to export and app with its data from the command line. Is there a way to do with budi cli or with the help of the budibase api and how? 
The purpose is that I would like to have a script taking a backup of an app periodically instead of having to do it manually from the backend.

**Comment by ConorWebb96** [comment, marked as answer]
Hey @gmentzik,

A breakdown of what you want can be found here: https://docs.budibase.com/docs/server-migration#exporting

The command you are looking for is here: `budi backups --export filename.tar.gz`

To explain this further depending on how you are hosting, you would likely need to setup a cron job that would run this command periodically.

I hope this helps!
